### DISCUSSION TITLE
Embedded Chartbrew with tenant division

### DISCUSSION THREAD

**Discussion by felipegs31** [chartbrew/chartbrew#155] [Ideas]
Hello Chartbew team,

I want to use your product with a connection to MongoDB.
I have a SaaS and my tenant division is logical on DB, so every collection document has a tenantId property.

Is there a way to divide this data using your product? So client A only sees his data on the embedded dashboard? And all automated? So when I create a new tenant (client) I don't have to manually change anything?

(Basically I am asking If you guys have row level security)

Thank you!

**Comment by razvanilin** [comment]
Hey @felipegs31 Currently there's no way to achieve this automatically. You would have to create the charts yourself and then embed them on their dashboard. You can make this process easier by using the "template" feature where you can replicate charts and just tweak the IDs (but still a manual process).

That said, this functionality will most likely be added to the product in Q1 2023 after the Datasets are re-worked to allow templating, dynamic variables, and API access

**Reply by ldurans** [reply]
@razvanilin is there already a way to do this?
